### PR TITLE
Reduce package size

### PR DESCRIPTION
**Pull request by yakushabb**
Needs to be tested carefully.

### CODE REVIEW

**Comment by yakushabb** [comment]
Package size might be redeced from 36.7MB to 16.4MB due to removing share/doc folder



image

image

**Comment by tsdgeos** [comment]
Why are you removing the application documentation?

**Comment by yakushabb** [comment]
Kate handbook opens the following page in the browser: https://docs.kde.org/stable5/en/kate/kate/index.html

Are the documentations visible offline?

**Comment by tsdgeos** [comment]
I guess that's a bug in our flatpak system [*], the normal/non-flathub kate opens the documentation locally.

[*] I guess the runtime should package khelpcenter

**Comment by yakushabb** [comment]
non-flatpak version from Fedora Rawhide also opens the same page.

Ekran Görüntüsü - 2024-06-07 02-03-59

**Comment by tsdgeos** [comment]
That's because you don't have khelpcenter installed

File: org.kde.kate.json [hunk 1]
```diff
@@ -15,6 +15,23 @@
...
+        "/share/metainfo/org.kde.konsole.appdata.xml",
+        "/share/metainfo/org.kde.markdownpart.metainfo.xml",
+        "/share/qlogging-categories6",
+        "/share/knotifications6",
+        "/share/zsh",
+        "/share/kio",
+        "/share/kconf_update",
+        "/share/man",
+        "/etc",
+        "/bin/kwrite",
```
Comment (travier) [inline org.kde.kate.json:32]: OK

File: org.kde.kate.json [hunk 2]
```diff
@@ -15,6 +15,23 @@
...
+    "cleanup": [
+        "/share/doc",
+        "/share/kateproject",
+        "/share/knsrcfiles",
+        "/share/metainfo/org.kde.kwrite.appdata.xml",
+        "/share/metainfo/org.kde.konsole.appdata.xml",
+        "/share/metainfo/org.kde.markdownpart.metainfo.xml",
+        "/share/qlogging-categories6",
+        "/share/knotifications6",
+        "/share/zsh",
```
Comment (travier) [inline org.kde.kate.json:27]: OK

File: org.kde.kate.json [hunk 3]
```diff
@@ -15,6 +15,23 @@
...
+        "/share/kio",
+        "/share/kconf_update",
+        "/share/man",
```
Comment (travier) [inline org.kde.kate.json:30]: OK

File: org.kde.kate.json [hunk 4]
```diff
@@ -15,6 +15,23 @@
         "--system-talk-name=org.freedesktop.UDisks2",
         "--talk-name=org.freedesktop.Flatpak"
     ],
+    "cleanup": [
+        "/share/doc",
+        "/share/kateproject",
+        "/share/knsrcfiles",
+        "/share/metainfo/org.kde.kwrite.appdata.xml",
+        "/share/metainfo/org.kde.konsole.appdata.xml",
+        "/share/metainfo/org.kde.markdownpart.metainfo.xml",
```
Comment (travier) [inline org.kde.kate.json:24]: OK, but that's super small.

**Review by travier — CHANGES_REQUESTED** [review]
If the docs don't work we should fix that. Some of the cleanups we can do, the others I don't think we can.

**Comment by yakushabb** [comment]
OK?
```
"/bin/kwrite"
"/share/zsh",
"/share/man",
```

Trivial

```
"/share/metainfo/org.kde.kwrite.appdata.xml",
"/share/metainfo/org.kde.konsole.appdata.xml",
"/share/metainfo/org.kde.markdownpart.metainfo.xml",
```

Not OK?

```
"/share/doc",
"/share/kateproject",
"/share/knsrcfiles",
"/share/qlogging-categories6",
"/share/knotifications6",
"/share/kio",
"/share/kconf_update",
"/etc",
"/lib/kconf_update_bin"
```

**Comment by travier** [comment]
Yes. Agree that the GitHub issues made than more confusing than I expected

**Comment by yakushabb** [comment]
Not impressive!

resim

**Review by travier — APPROVED** [review]

**Comment by tsdgeos** [comment]
ok for me too i guess

**Comment by bbhtt** [comment]
bot, build

**Review by travier — APPROVED** [review]

**Pull request merged**